### PR TITLE
K8SPSMDB-265 cross region replication docs

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Review by egegunes — CHANGES_REQUESTED** [review]

File: source/replication.rst [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+should not happen if we want Operator to run the cluster as *Replica*, so first
+of all the cluster should be put into unmanaged state by setting the
+``unmanaged`` key in the ``deploy/cr.yaml`` configuration file to true.
+
+.. note:: Setting ``unmanaged`` to true will not only prevent the Operator from
+   controlling the Replica Set configuration, but it will also result in not
+   generating certificates and users credentials for new clusters.
+
+The cluster should also "know" about external nodes for the Replica Sets. You
+can configure this information in the ``replsets.externalNodes`` and
```
Comment (spron-in) [inline source/replication.rst:157]: know and information are killing the vibe

Comment (spron-in) [reply source/replication.rst:157]: user does not need to add external nodes on replica at all. not for config servers, not for replsets.

File: source/replication.rst [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+     unmanaged: true
+     replsets:
+     - name: rs0
+       size: 3
+       externalNodes:
+       - host: rs0-repl0.percona.com
+         port: 27017
+         priority: 0
+         votes: 0
+       - host: rs0-repl1.percona.com
```
Comment (spron-in) [inline source/replication.rst:189]: this is not needed on replica.

File: source/replication.rst [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+       ...
+     sharding:
+       configsvrReplSet:
+       size: 3
+       externalNodes:
+         - host: rs0-repl0.percona.com
+           port: 27017
+           priority: 0
+           votes: 0   
+         - host: rs0-repl1.percona.com
```
Comment (spron-in) [inline source/replication.rst:199]: not needed on replica

File: source/replication.rst [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+         ...
+
+*Main* and *Replica* cluster should have same Secrets objects, so don't forget
+to apply Secrets from your *Main* cluster. Names of the corresponding objects
+are set in the ``users``, ``ssl``, and ``sslInternal`` keys of the Custom
+Resource ``secrets`` subsection (``my-cluster-name-secrets``,
+``my-cluster-name-ssl``, and ``my-cluster-name-ssl-internal`` by default).
```
Comment (spron-in) [inline source/replication.rst:206]: ah, I see it now. Lets move it to corresponding section?

**Review by spron-in — CHANGES_REQUESTED** [review]

File: source/replication.rst [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+
+The above example is using the LoadBalancer Kubernetes Service object, but there
+are other options (ClusterIP, NodePort, etc.).
+
+.. note:: The above example will create a LoadBalancer per each MongoDB Pod.
+   In most cases, this Load Balancer should be internet-facing for cross-region
+   replication to work.
+   
+To list the endpoints assigned to Pods, list the Kubernetes Service objects by 
+executing ``kubectl get services -l "app.kubernetes.io/instance=CLUSTER_NAME"`` command.
```
[the remark this answers is not in the document]
Comment (fiowro) [reply source/replication.rst:65]: yes)

**Pull request merged**